contracts/permissionless/Orders.sol

```diff
@@ -116,16 +114,6 @@ contract Orders is Withdrawable, Utils2, OrdersInterface {
         onlyAdmin
         returns (bool)
     {
-        bool hintIsCurrentPosition = false;
-        bool hintIsRightPositionAfterUpdate = false;
-        if (hintIsCurrentPosition && hintIsRightPositionAfterUpdate) {
-            // Order is in the right position, update amounts
-            orders[orderId].srcAmount = srcAmount;
-            orders[orderId].dstAmount = dstAmount;
-            AmountUpdateOnly();
-            return true;
-        }
-
         if (isRightPosition(srcAmount, dstAmount, prevId)) {
```

Comment:
if the order doesn't change position here the test might fail while still the correct position.
I know in our design it shouldn't happen. but still. its a flow in the code.
I can build a test that will prove it....

Reply:
I mean. the correct test will make this update call fail...

Reply:
Agree, added test and fixed the issue.